src/preset.ts

```diff
@@ -1,7 +1,7 @@
 import path from 'path'
 import * as tsup from 'tsup'
 import * as esbuild from 'esbuild'
-import { solidPlugin } from 'esbuild-plugin-solid'
+import { type Options, solidPlugin } from 'esbuild-plugin-solid'
```

Comment:
Please alias Options or the while import as solid so it's obvious what Options is.